Source/Cake.AddinDiscoverer/exclusionlist.json

```diff
@@ -41,6 +41,9 @@
     "Build",
     "Cake",
     "Cake-Contrib",
-    "Script"
+    "Script",
+    "Cake-Addin",
```

Comment:
LGTM, though @Jericho needs to double-check the exclusion is not case sensitive because existing addins are using the tags all in lowercase e.g. `cake-addin` instead of `Cake-Addin`.

Reply:
I checked - saw the discussion in the other PR 😄

Reply:
Tag exclusion is indeed case-insensitive.